internal/output/plain.go

```diff
@@ -320,8 +320,7 @@ func sprintTable(slice []interface{}) (string, error) {
 	// Don't print whitespace lines
 	t.SetHideLines([]string{"betweenLine", "top", "aboveTitle", "LineTop", "LineBottom", "bottomLine"})
 	t.SetAlign("left")
-
-	render := t.Render("simple")
+	render := t.Render("standard")
```

Comment:
Rather than bake our layout into tabulate and marry us to this fork, why not instead fix the issue that stopped you from using the original and open a PR? We can use the fork in the interim.

So in other words: export the fields that prevented you from creating our own custom layout.

Reply:
Fair point. The library appears to be fairly stale but I think this is from lack of issues and PRs, not neglect from the author so I will create a PR there with the changes we need.